Jumpscale/clients/blockchain/tfchain/README.md

```diff
@@ -684,3 +684,44 @@ unconfirmed  Tx: 573290763024ae0a5e981412598a3d41bc02f8da628fa1e1adfe07d98818c68
         > to: 0125c0156f6c1c0bc43c7d38e17f8948300564bef63caac05c08b0fd68996e494704bbbe0268cb
         > from: 01773a1dd123347e1030f0822cb8d22082fe3f9b0ea8563d4ac8e7abc377eba920c47efb2fd736
 ```
+
+### Capacity reservation
+
+During the fist beta phase of the public launch of the TF grid, beta tester will be able to reserve 2 kind of workload on the grid.
+
+- Zero-OS virtual machines
+- S3 archive storage instances
+
+**At the time of writing, everything happens on the testnet network**
+**Don't send real TFT from the main network !!**
+
+#### How to reserve some capacity on the Threefold Grid
+
+To be able to make a reservation you first need to:
+
+- have a wallet with some fund
```

Comment:
```suggestion
- have a wallet with sufficient funds
```
Some is wrong, as `0.000001` TFT is also some funds, but certainly won't do you any good